docs/content/contributing-c.md

```diff
@@ -0,0 +1,7 @@
+# Contributing C
+
+Run `./format_code.sh` before committing to format the code.
+
+## Code Quality
+
+Code in the directory `lcmgen` leaks memory. It is not an uncommon practice to allow this for short lived CLI tools. 
```

Comment:
That's interesting. I'd still prefer to treat memory leaks as bugs but I'd consider the current leaks to be a low priority bug since no one has complained about them causing issues. I'm also fine if contributions have bugs with similar a similar level of impact. We can always open a new issue for those.